src/LieTheory/WeylGroup.jl

```diff
@@ -753,6 +753,43 @@ function reflection(beta::RootSpaceElem)
   return W([list_of_indices; current_index; reverse(list_of_indices)])
 end
 
+###############################################################################
+# G-set functionality
+
+function gset_by_type(W::WeylGroup, Omega, ::Type{RootSpaceElem}; closed::Bool=false)
+  return GSetByElements(W, *, Omega; closed=closed, check=false)
+end
+
+function gset_by_type(W::WeylGroup, Omega, ::Type{WeightLatticeElem}; closed::Bool=false)
+  return GSetByElements(W, *, Omega; closed=closed, check=false)
+end
+
+function action_homomorphism(Omega::GSetByElements{WeylGroup,S}) where {S}
+  W = acting_group(Omega) # our base group
+
+  # Compute a permutation group `G` isomorphic with `W`.
+  phi = isomorphism(PermGroup, W)
```

Comment:
This function is currently experimental. I'll provide a PR moving it to source, so we can use it here

Reply:
Looking again at this, I remember that this code stays experimental until we have Group morphisms that support WeylGroups.
Could you thus please move the changes in `src/LieTheory/WeylGroup.jl` to `experimental/LieAlgebras/src/WeylGroup.jl` and the respective tests to `experimental/LieAlgebras/test/WeylGroup-test.jl`?